package.json

```diff
@@ -38,11 +41,12 @@
     "@types/js-yaml": "^4.0.5",
     "@vitest/coverage-v8": "^1.2.1",
     "mkdirp": "^2.1.3",
+    "openapi-typescript-codegen": "^0.27.0",
```

Comment:
I guess you don't need the code generator anymore

Reply:
resolved in https://github.com/redhat-developer/podman-desktop-redhat-account-ext/pull/35